jena-arq/src/test/java/org/apache/jena/riot/tokens/TestTokenizerText.java

```diff
@@ -25,911 +25,938 @@
-        tokenizeAndTestExact("123", TokenType.INTEGER, "123") ;
+    public void tokenUnit_num01() {
+        tokenizeAndTestExact("123", TokenType.INTEGER, "123");
     }
 
     @Test
-    public void tokenUnit_num2() {
+    public void tokenUnit_num02() {
         // This is a change in Turtle (and SPARQL 1.1)
-        tokenizeAndTestFirst("123.", TokenType.INTEGER, "123") ;
+        tokenizeAndTestFirst("123.", TokenType.INTEGER, "123");
     }
 
     @Test
-    public void tokenUnit_num3() {
-        tokenizeAndTestExact("+123.456", TokenType.DECIMAL, "+123.456") ;
+    public void tokenUnit_num03() {
+        tokenizeAndTestExact("+123.456", TokenType.DECIMAL, "+123.456");
     }
 
     @Test
-    public void tokenUnit_num4() {
-        tokenizeAndTestExact("-1", TokenType.INTEGER, "-1") ;
+    public void tokenUnit_num04() {
+        tokenizeAndTestExact("-1", TokenType.INTEGER, "-1");
     }
 
     @Test
-    public void tokenUnit_num5() {
-        tokenizeAndTestExact("-1e0", TokenType.DOUBLE, "-1e0") ;
+    public void tokenUnit_num05() {
+        tokenizeAndTestExact("-1e0", TokenType.DOUBLE, "-1e0");
     }
 
     @Test
-    public void tokenUnit_num6() {
-        tokenizeAndTestExact("1e+1", TokenType.DOUBLE, "1e+1") ;
+    public void tokenUnit_num06() {
+        tokenizeAndTestExact("1e+1", TokenType.DOUBLE, "1e+1");
     }
 
     @Test
-    public void tokenUnit_num7() {
-        tokenizeAndTestExact("1.3e+1", TokenType.DOUBLE, "1.3e+1") ;
+    public void tokenUnit_num07() {
+        tokenizeAndTestExact("1.3e+1", TokenType.DOUBLE, "1.3e+1");
     }
 
     @Test
-    public void tokenUnit_num8() {
-        tokenizeAndTestFirst("1.3.4", TokenType.DECIMAL, "1.3") ;
+    public void tokenUnit_num08() {
+        tokenizeAndTestFirst("1.3.4", TokenType.DECIMAL, "1.3");
     }
 
     @Test
-    public void tokenUnit_num9() {
-        tokenizeAndTestFirst("1.3e67.7", TokenType.DOUBLE, "1.3e67") ;
+    public void tokenUnit_num09() {
+        tokenizeAndTestFirst("1.3e67.7", TokenType.DOUBLE, "1.3e67");
     }
 
     @Test
     public void tokenUnit_num10() {
-        tokenizeAndTestExact(".1", TokenType.DECIMAL, ".1") ;
+        tokenizeAndTestExact(".1", TokenType.DECIMAL, ".1");
     }
 
     @Test
     public void tokenUnit_num11() {
-        tokenizeAndTestExact(".1e0", TokenType.DOUBLE, ".1e0") ;
+        tokenizeAndTestExact(".1e0", TokenType.DOUBLE, ".1e0");
     }
 
     @Test
     public void tokenUnit_num12() {
-        // This is not a hex number.
+        tokenizeAndTestExact("+.1", TokenType.DECIMAL, "+.1");
+    }
 
-        Tokenizer tokenizer = tokenizeAndTestFirst("000A     .", TokenType.INTEGER, "000") ;
-        testNextToken(tokenizer, TokenType.KEYWORD, "A") ;
+    @Test
+    public void tokenUnit_num13() {
+        tokenizeAndTestExact("-.1", TokenType.DECIMAL, "-.1");
+    }
+
+    @Test
+    public void tokenUnit_num14() {
+        tokenizeAndTestExact("+.1e0", TokenType.DOUBLE, "+.1e0");
```

Comment:
:+1: do we have/need a test like this, but without the sign too?

Reply:
`tokenUnit_num11` tests ".1e0".